client/src/components/v2/CameraRecordingStatus.js

```diff
@@ -0,0 +1,93 @@
+import React, { useCallback, useState } from 'react';
+import { useChannel, emit } from 'api/v2/socket';
+import formatBytes from 'utils/formatBytes';
+// import { Card } from 'react-bootstrap';
+import PropTypes from 'prop-types';
+
+/**
+ * @typedef {object} CameraRecordingStatusProps
+ * @property {'primary'|'secondary'} device Camera screen
+ */
+
+/**
+ *  Status of a camera
+ *
+ * @param {CameraRecordingStatusProps} props Props
+ * @returns {React.Component<CameraRecordingStatusProps>} Component
+ */
+export default function CameraRecordingStatus({ device }) {
+  /**
+   * Format a camera recording status payload into JSX
+   *
+   * @param {string} payload Payload structure is defined in https://www.notion.so/V3-MQTT-Topics-66e6715d0e1941ffaa82020e5868fbae. See /v3/camera/recording/status/<primary/>secondary>
+   * @returns {React.Component} Component
+   */
+  function formatPayload(payload) {
```

Comment:
This whole section is a bit confusing to me because it feels like it's trying to do multiple things at once and I just get confused? See below on how I might improve this.

Reply:
See https://github.com/monash-human-power/dashboard/pull/74#discussion_r461242513